projects/lib/src/chessgame.h

```diff
@@ -137,6 +142,7 @@ class LIB_EXPORT ChessGame : public QObject
 		QSemaphore m_pauseSem;
 		QSemaphore m_resumeSem;
 		GameAdjudicator m_adjudicator;
+		const Tournament* m_tournament;
```

Comment:
I'll have to think about this a little. I'd like to avoid the two-way dependency between `ChessGame` and `Tournament`.